Widgets/Widgets.swift

```diff
@@ -249,6 +249,17 @@ struct Widgets: WidgetBundle {
     }
 }
 
+struct VPNBundle: WidgetBundle {
+    @WidgetBundleBuilder
+    var body: some Widget {
+        if #available(iOS 17, *) {
+            VPNStatusWidget()
+            VPNSnoozeLiveActivity()
+        }
+    }
+}
```

Comment:
The widget builder syntax only allows up to 10 elements, and the addition of the Live Activity brings up to 11, so I'm working around this with a widget bundle.